opensrp-app/src/main/java/org/smartregister/sync/ClientProcessorForJava.java

```diff
@@ -648,7 +643,7 @@ public void updateClientDetailsTable(Event event, Client client) {
                     attributes.put(key, value.toString());
                 }
             }
-        } catch (Exception e) {
+        } catch (NullPointerException e) {
```

Comment:
Is this the only the exception that can occur

Reply:
yeah i think so